tests/test_state/test_remote_state_manager.py

```diff
@@ -30,36 +33,61 @@
 
 
 @pytest.fixture
-def remote_state_config() -> RemoteStateConfig:
-    """Fixture for a remote state configuration.
+def mocked_matcha_config_json_object() -> Dict[str, Dict[str, str]]:
```

Comment:
To avoid duplication, this could be put into the `conftest.py` as it's defined twice in the test suite. In fact, all of the fixtures below could be.

Reply:
All added to conftest, thanks Jon.